source/frontend/src/features/mapSideBar/compensation/detail/CompensationRequisitionDetailContainer.test.tsx

```diff
@@ -40,6 +40,7 @@ vi.mock('@/hooks/repositories/useRequisitionCompensationRepository', () => ({
   useCompensationRequisitionRepository: () => {
     return {
       getCompensationRequisitionProperties: mockGetCompReqPropertiesApi,
+      getCompensationRequisitionPayees: mockGetCompReqPropertiesApi,
```

Comment:
getCompensationRequisitionPayees is using the same mocked function as getCompensationRequisitionProperties - please add separate mock for it